packages/connect-web/src/connect-transport.ts

```diff
@@ -102,6 +103,13 @@ export interface ConnectTransportOptions {
    * Options for the binary wire format.
    */
   binaryOptions?: Partial<BinaryReadOptions & BinaryWriteOptions>;
+
+  /**
+   * Controls whether or not the fetch client should use Connect GET
```

Comment:
Can you avoid "fetch client" here and in transport-options.ts? 

For example "Controls whether or not Connect GET requests should be used when ...".

Since Node.js does not use the fetch API fetch(), it could be a bit confusing otherwise.